### DISCUSSION TITLE
Where are the pre-built (Win32) binaries for 0.0.24?

### DISCUSSION THREAD

**Discussion by garoto** [uutils/coreutils#5915] [General]
Title. Can't see anything in the README regarding a change of policy on releasing binaries.

**Comment by tertsdiepraam** [comment]
We've been wondering this ourselves too. They haven't been generated by GitHub Actions as they should have been.

**Reply by tertsdiepraam** [reply]
I'll add that we usually recommend getting uutils through a package manager if possible, so the package manager can do some additional setup (like making symlinks to the multicall binary).
